spikeinterface/sortingcomponents/benchmark/benchmark_clustering.py

```diff
@@ -179,15 +191,91 @@ def run(self, peaks=None, positions=None, method=None, method_kwargs={}):
         self.gt_labels = self.gt_sorting.to_spike_vector()['unit_ind']
 
 
+    def _get_colors(self, sorting, excluded_ids=[-1]):
```

Comment:
excluded_ids is not a good naming.
I would use something like `force_black_for=[-1]`

Reply:
forget it.

Reply:
Too late, I changed the name